webview/src/experiments/components/table/ExperimentCell.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react'
+import cx from 'classnames'
+import { VSCodeTag } from '@vscode/webview-ui-toolkit/react'
+import { CommitData } from 'dvc/src/experiments/webview/contract'
+import styles from './styles.module.scss'
+import Tooltip from '../../../shared/components/tooltip/Tooltip'
+import { Icon } from '../../../shared/components/Icon'
+import { GitCommit } from '../../../shared/components/icons'
+
+const CommitCell: React.FC<{
+  commit: CommitData
+  description: string
+  label: string
+  sha?: string
+}> = ({ commit, description, label, sha }) => {
+  const { tags, author, message, date } = commit
+  const tooltipContent = (
+    <div className={styles.experimentCellSecondaryNameTooltip}>
+      <div>
+        <p className={styles.sha}>
+          <Icon width={16} height={16} icon={GitCommit} /> {sha?.slice(0, 7)}
+        </p>
+        {tags.length > 0 && (
+          <>
+            {tags.map(tag => (
+              <VSCodeTag className={styles.tag} key={tag}>
+                {tag}
+              </VSCodeTag>
+            ))}
+          </>
+        )}
+      </div>
+      <p>
+        {author}, {date}
+      </p>
+      <p className={styles.message}>{message}</p>
+    </div>
+  )
+
+  return (
+    <Tooltip
+      placement="bottom-start"
+      appendTo={document.body}
+      content={tooltipContent}
+    >
+      <div className={styles.experimentCellText}>
+        <span>{label}</span>
+        <br />
+        <span className={styles.experimentCellSecondaryName}>
+          <Icon width={14} height={14} icon={GitCommit} />{' '}
+          <span>{description}</span>
+        </span>
+      </div>
+    </Tooltip>
+  )
+}
+
+type ExperimentCellProps = {
+  commit?: CommitData
+  description?: string
+  error?: string
+  label: string
+  sha?: string
+}
+
+export const ExperimentCell: React.FC<ExperimentCellProps> = ({
+  commit,
+  description,
+  error,
+  label,
+  sha
+}) => {
+  if (!description) {
+    return (
+      <div className={styles.experimentCellText}>
+        <span>{label}</span>
+      </div>
+    )
+  }
+
+  return commit ? (
+    <CommitCell
+      commit={commit}
+      description={description}
+      label={label}
+      sha={sha}
+    />
+  ) : (
+    <div className={styles.experimentCellText}>
+      <span
```

Comment:
I was staring at the table for like ten minutes, trying to figure out why it looked so off to me when it finally hit me that the commit rows have secondary text second while the experiments have their secondary text first. 

I'm assuming most people won't notice 😅, but maybe we should keep secondary text second across all rows? Example of how this would look:

<img width="511" alt="image" src="https://user-images.githubusercontent.com/43496356/233399741-f33e6bcb-123d-4dfc-b764-b59d3cd61d73.png">

Technically, in design, you can have secondary text first depending on other factors but the main reason that the rows were bothering me is the fact we have a mixture of secondary being first and second. Though again, I'm probably the only person getting bothered :)

Reply:
I know that this breaks design principles. The reason that we've arrived here is that previously experiment names were less useful (e.g `exp-43234`) and the sha was thought of as the primary information identifier. The DVC team tried to solve the problem of not being able to navigate a lot of experiments by making the name easier to read/more memorable. The name as since become the primary identifier. However, the CLI still displays the information in the same way:

<img width="414" alt="image" src="https://user-images.githubusercontent.com/37993418/233486120-356ed505-7961-4324-a5cd-1044ffcf28e0.png">

Studio have addressed this problem in the following way:

<img width="585" alt="image" src="https://user-images.githubusercontent.com/37993418/233484304-b85fa6ba-9e53-4b3f-be6c-0f091dc86f9d.png">

For more context please read https://github.com/iterative/vscode-dvc/issues/3619#issuecomment-1510507639 & https://github.com/iterative/vscode-dvc/issues/3619#issuecomment-1510514599 but the tl;dr is - we display this information in a lot of places, we should be standardising between the products, having the name first is a better option but a much larger change across multiple products.

Reply:
Oh, ok, makes sense! I didn't realize we were interested in matching order as well as focus across the extension! Hmmm, in that case, we could make secondary text first across all the rows: 

<img width="430" alt="image" src="https://user-images.githubusercontent.com/43496356/233494584-33ccda5c-1f82-4d60-84c2-4e6d2d317dbe.png">

Or just leave as is, up to you!